doc-content/enterprise-only/openshift/template-deploy-ha-resources-proc.adoc

```diff
@@ -0,0 +1,24 @@
+[id='template-deploy-ha-resources-{context}-proc']
+= Configuring resource usage for a high-availability deployment
+
+If you are deploying the high-availability template (`{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-authoring-ha.yaml`), you can optionally configure resource usage to optimize performance for your requirements.
```

Comment:
More information about memory sizing can be found in OpenShift documentation [https://docs.openshift.com/container-platform/3.11/dev_guide/application_memory_sizing.html](https://docs.openshift.com/container-platform/3.11/dev_guide/application_memory_sizing.html)

Reply:
Not required, but can be easier for users to understand more memory sizing.

Reply:
To learn more about memory sizing, see (link).

Reply:
I see you have not included the more information line. Is this deliberate?